charts/gardener-extension-os-ubuntu/values.yaml

```diff
@@ -5,6 +5,15 @@ image:
 
 resources: {}
 
+vpa:
```

Comment:
You need to run `make generate` to have this change reflected also in `example/controller-registration.yaml` (as this is a change under `charts/gardener-extension-os-ubuntu/`).

Reply:
i updated the `example/controller-registration.yaml`